content/about/tob.md

```diff
@@ -39,10 +39,11 @@ The TOB holds infrequent meetings which are usually recorded and posted on [the
 
 {{< card-container >}}
 
-{{< card img="/img/tob/Tycho-Andersen.jpeg" name="Tycho Andersen" termdates="2021-01-29 → 2023-01-29" company="Netflix" github="https://github.com/tych0" >}}
+{{< card img="/img/tob/Brandon-Mitchell.jpeg" name="Brandon Mitchell" termdates="2023-01-29 → 2025-01-29" company="Independent" github="https://github.com/sudo-bmitch" >}}
+Brandon is an open source developer working in the container ecosystem.
 {{< /card >}}
 
-{{< card img="/img/tob/Vincent-Batts.jpeg" name="Vincent Batts" termdates="2021-01-29 → 2023-01-29" company="Kinvolk" github="https://github.com/vbatts" linkedin="https://www.linkedin.com/in/vincentbatts/" twitter="https://twitter.com/vbatts" >}}
+{{< card img="/img/tob/Vincent-Batts.jpeg" name="Vincent Batts" termdates="2021-01-29 → 2025-01-29" company="Kinvolk" github="https://github.com/vbatts" linkedin="https://www.linkedin.com/in/vincentbatts/" twitter="https://twitter.com/vbatts" >}}
```

Comment:
(multiple occurrences) a term is 2 years, so the term start date should also be updated for TOB members who were re-elected

```suggestion
{{< card img="/img/tob/Vincent-Batts.jpeg" name="Vincent Batts" termdates="2023-01-29 → 2025-01-29" company="Kinvolk" github="https://github.com/vbatts" linkedin="https://www.linkedin.com/in/vincentbatts/" twitter="https://twitter.com/vbatts" >}}
```

Reply:
Fixed. Thanks for catching that!